infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/ReferenceLineService.kt

```diff
@@ -137,6 +138,12 @@ class ReferenceLineService(
         return dao.fetchVersion(publishType, trackNumberId)?.let(dao::fetch)
     }
 
+    fun getByTrackNumberAtMoment(publishType: PublishType, trackNumberId: IntId<TrackLayoutTrackNumber>, moment: Instant): ReferenceLine? {
```

Comment:
Mihinkäs tätä oikeastaan käytetään? Tuntuu hiukan sellaiselta operaatiolta että olisi toivottavampaa löytää oikea line jotain muuta kautta.

Reply:
Mun viimeisimpien muutosten (ei vielä puskettu) myötä tätä ei käytetä itse asiassa enää mihinkään. Hyvä huomio, poistan tän. Käytännössä tässä taisi olla taustalla se ajatus, että tietomalli mahdollistaa, että a) teoriassa ratanumeroon liittyvä pituusmittauslinjainstanssi voi vaihtua kokonaan ja b) vaikka se ei vaihtuisi, niin ratanumero ja pituusmittauslinja voidaan julkaista toisistaan erillään. Tämä lienee ollut käytössä ratanumeron tietojen haussa. Ajattelin, että operaattorin kannalta olisi vähiten epäselvää jos ratanumeron muutoksissa (joissa tarvitaan myös pituusmittauslinjaa) vertailtava edellisen julkaisun pituusmittauslinja olisi aina ratanumeron edellisen muutoshetken pituusmittauslinja (joka siis teoriassa voi olla eri kuin vertailtavan julkaisun pituusmittauslinja, mutta vaan version-1.) Toki tällöin se ei kuvasta enää täysin pelkästään ka. julkaisussa tehtyjä muutoksia, joten täytyy ehkä ottaa muutenkin lisää mietintää näiden alkuperäisten vaatimusten osalta 🤔